index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+<head>
+	<title>Hannah Golightly</title>
+	<link rel="stylesheet" type="text/css" href="normalize.css" />
+	<link rel="stylesheet" type="text/css" href="style.css" />
+	<link href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+	<meta name="viewport" content="width=device-width, initial-scale=1" charset="utf-8" >
```

Comment:
charset should be in a separate meta tag. So you should have two like this:

`<meta name="viewport" content="width=device-width, initial-scale=1">`
`<meta  charset="utf-8">`